milvus/question_answering.py

```diff
@@ -0,0 +1,97 @@
+import os
+import pandas as pd
+import numpy as np
+import csv
+import gradio
+from pymilvus import connections, FieldSchema, CollectionSchema, DataType, Collection, utility
+from towhee import pipe, ops
+from towhee.datacollection import DataCollection
+
+# Milvus parameters
+host = os.environ["MILVUS_HOST"]
+port = os.environ["MILVUS_PORT"]
```

Comment:
Not a code review per se, but maybe you could set a default value in the case the env vars doesn't exist, or, just give a error message.